public/assets/js/scripts.js

```diff
@@ -132,6 +133,14 @@ $(document).ready(function () {
     $(this).find('.faq-icon, .faq-option-category-icon').text((text) => text === '+' ? '-' : '+');
   });
 
+  $('.recommendation-card').on('shown.bs.collapse', function () {
```

Comment:
I think this can also be achieved with CSS. When the accordion is collapsed/ closed, the corresponding `a` tag has a class name of `collapse`. This class is removed when the accordion is opened. Using CSS, target the span you include and let it load the appropriate image when collapsed and when open.

Reply:
was unable to replicate this tho. maybe I'm doing something wrong